src/js/redux/index.js

```diff
@@ -80,6 +82,8 @@ const reducers = {
   [POPULATE_QUICKSTARTS_CATALOG]: populateQuickstartsReducer,
   [DISABLE_QUICKSTARTS]: disableQuickstartsReducer,
   [UPDATE_DOCUMENT_TITLE_REDUCER]: documentTitleReducer,
+  'toggle-notifications-drawer': notificationsDrawerReducer,
+  'add-notification': addNewNotificationReducer,
```

Comment:
Can we make these into constants?

Reply:
Definitely. This is just a draft to demo the features. Do not take anything seriously, only the concept.